packages/web/server/queries/complex/pools/market.ts

```diff
@@ -23,7 +23,8 @@ export function getCachedPoolMarketMetricsMap(): Promise<
   return cachified({
     cache: metricPoolsCache,
     key: "pools-metrics-map",
-    ttl: 1000 * 60 * 5, // 5 minutes
+    ttl: 1000 * 10, // 10 seconds
+    staleWhileRevalidate: 1000 * 25, // 25 seconds, edge function run time limit
```

Comment:
Why are we lowering ttl to 10 seconds?

I think that it isn't really useful to update the volume data frequently. Infrequent update > 1 min makes sense to me. Reducing the TTL would increase the load on the data backend.

The `stalteWhileRevalidate` addition makes sense but I don't think that it would lower the load on the data backend

Reply:
I think I'm going to use a remote cache and increase this value to 1 min, 1 hour respectively.

I have it 10s, 25s here because edge funcitons only live for 25s, meaning the in memory cache only lives for 25s at most. So, for 10s users will hit the cache, and the remaining 15s users will still hit cache and data will be updated in background.

Reply:
The 1 min ttl makes sense. What is the reason for choosing such a high number for stale while revalidate? Wouldn't it be more than we need?